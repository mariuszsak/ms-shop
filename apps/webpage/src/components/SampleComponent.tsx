import React from 'react';

const SampleComponent = () => {
    return (
        <div>
            <h1>test</h1>
        </div>
    );
};

export default SampleComponent;