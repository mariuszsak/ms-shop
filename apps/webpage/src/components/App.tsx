import React from 'react';
import SampleComponent from "./SampleComponent";

function App() {
    return (
        <div>
            <SampleComponent/>
        </div>
    );
}

export default App;