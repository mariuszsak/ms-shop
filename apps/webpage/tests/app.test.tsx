import {shallow} from "enzyme";
import App from "../src/components/App";
import SampleComponent from "../src/components/SampleComponent";

describe('Initial test', () => {
    it('Should render correctly', () => {
        const component = shallow(<App/>)
        expect(component).toMatchSnapshot();
    });
    it('Should render an component correctly', () => {
        const component = shallow(<SampleComponent/>)
        expect(component).toMatchSnapshot();
    })
})