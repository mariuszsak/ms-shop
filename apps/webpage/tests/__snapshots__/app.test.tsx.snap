// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Initial test Should render an component correctly 1`] = `
<div>
  <h1>
    test
  </h1>
</div>
`;

exports[`Initial test Should render correctly 1`] = `
<div>
  <SampleComponent />
</div>
`;
